docs/reference/query-language.md

```diff
@@ -35,6 +35,10 @@ Quickwit supports parenthesis to group multiple clauses:
 (color:red OR color:green) AND size:large
 ```
 
+### Slop Operator
```

Comment:
We should document that this feature requires recording positions and link to the position option.